demos/guide/quic-hq-interop.c

```diff
@@ -607,11 +609,32 @@ static size_t build_request_set(SSL *ssl)
 
         /*
          * We don't strictly have to do this check, but our quic client limits
-         * our max data streams to 100, so we're just batching in groups of 100
+         * our max data streams to 25, so we're just batching in groups of 100
```

Comment:
Slightly confused by the 25 earlier on the line and then 100 later

Reply:
inconsistency is the spice of life :)